query_server/query/src/metadata.rs

```diff
@@ -166,6 +176,183 @@ impl MetaData for LocalCatalogMeta {
         Ok(())
     }
 
+    fn show_database(&self) -> Result<Output> {
```

Comment:
Would it be better to use `show_databases`  instead of `show_database`

Reply:
nice hint